uniffi_bindgen/src/library_mode.rs

```diff
@@ -128,11 +137,26 @@ fn find_components(
                 ci.add_metadata(metadata)?;
             };
             ci.add_metadata(group)?;
-            Ok(ci)
+            let config = config_supplier
+                .get_toml(ci.crate_name())?
+                .unwrap_or_default();
+            Ok(Component { ci, config })
         })
         .collect()
 }
 
+/// Run [find_components] using cargo-metadata to find UDL and TOML files
+#[cfg(feature = "cargo-metadata")]
+pub fn find_components_cargo(library_path: &Utf8Path) -> Result<Vec<Component<TomlTable>>> {
```

Comment:
Is there any way to push this out to the caller? I'm thinking of the --no-deps issue and how a user with that problem would avoid it here?

Reply:
I can think of a couple options, what do you think is best?
 - Adding add a `no_deps` flag
 - Not exposing this for now and forcing users to use the `find_components` function.  If we get requests to support it, we can ask those users what API they want.

Reply:
I was thinking that this entire function just move to the caller (which is the gecko bindings?) - so yeah, only exposing `find_components` and leave the job of running cargo-metadata to the callers, like it is now in the cli and tests.

Reply:
Sounds good, just pushed an update that removes `find_components_cargo`.